packages/app-desktop/gui/ConfigScreen/FontSearch.tsx

```diff
@@ -0,0 +1,90 @@
+import React = require('react');
+import { useMemo, useState } from 'react';
+import { CSSProperties } from 'styled-components';
+
+const searchListStyle: CSSProperties = {
+	backgroundColor: 'var(--joplin-background-color)',
+	maxHeight: '200px',
+	minHeight: '10px', // Needed to prevent the list from collapsing when empty
+	width: '50%',
+	minWidth: '20em',
+	overflowY: 'auto',
+	border: '1px solid #ccc',
+	borderRadius: '5px',
+	display: 'none',
+	marginTop: '10px',
+};
+const optionStyle: CSSProperties = {
+	padding: '5px',
+	borderBottom: '1px solid #ccc',
+	cursor: 'pointer',
+};
+
+const FontSearch = (props: any) => {
+	const { _key: key, updateSettingValue, inputType, inputStyle, settings, fonts } = props;
```

Comment:
Only pass what you need, and you don't need all settings here

Reply:
fixed with 9832a9e